### PR TITLE
🐞 fix: cmaketools can't delect preset

### PR DESCRIPTION
**Pull request by gukazma**
vscode plugin ``cmake tools`` can't delect ``CMakePresets.json`` that contains:
```
"$schema": "https://cmake.org/cmake/help/latest/_downloads/3e2d73bff478d88a7de0de736ba5e361/schema.json"
"toolchainFile": "${sourceDir}/vcpkg/scripts/buildsystems/vcpkg.cmake"
```
I have deleted and modified these two lines to ensure that the plugin can run properly

### CODE REVIEW

File: CMakePresets.json [hunk 1]
```diff
@@ -13,7 +12,9 @@
             "description": "Configure with vcpkg toolchain and generate Ninja project files for all configurations",
             "binaryDir": "${sourceDir}/builds/${presetName}",
             "generator": "Ninja Multi-Config",
-            "toolchainFile": "${sourceDir}/vcpkg/scripts/buildsystems/vcpkg.cmake"
```
Comment (lukka) [inline CMakePresets.json:16]: CMakePresets.json v3 and up should support the `toolchainFile` property, could you please double check?

File: CMakePresets.json [hunk 2]
```diff
@@ -1,5 +1,4 @@
 {
-    "$schema": "https://cmake.org/cmake/help/latest/_downloads/3e2d73bff478d88a7de0de736ba5e361/schema.json",
```
Comment (lukka) [inline CMakePresets.json:2]: Right, CMake-Tools is supporting CMakePresets.json version 6, but the $schema attribute was introduced in version 8.

File: CMakePresets.json (hunk 1 above)
Comment (gukazma) [reply CMakePresets.json:16]: But cmake-tools plugin recognizes CMakePresets.json with toolchainFile attribute

Comment (gukazma) [reply CMakePresets.json:16]: 但是cmake-tools插件不识别带toolchainFile属性的CMakePresets.json

Comment (gukazma) [reply CMakePresets.json:16]: But the ``cmake-tools`` plugin does not recognize ``CMakePresets.json`` with ``toolchainFile`` attribute

Comment (lukka) [reply CMakePresets.json:16]: @gukazma works for me on CMake Tools `v1.17.17`.

The `$schema` option needs to be dropped from the CmakePresets.json, but the `toolchainFile` works just fine, here the content of the Output->CMakeTools pane (I enabled verbose output in `setttings.json` with `"cmake.loggingLevel": "debug"`):

```
[expand] expanding cmake
[main] Configuring project: CppCMakeVcpkgTemplate 
[main] Saving open files before configure/build
[driver] Removing /Users/user/github/CppCMakeVcpkgTemplate/builds/ninja-multi-vcpkg/CMakeCache.txt
[driver] Removing /Users/user/github/CppCMakeVcpkgTemplate/builds/ninja-multi-vcpkg/CMakeFiles
[expand] expanding /Users/user/github/CppCMakeVcpkgTemplate
[driver] Start configure 
[driver] Running pre-configure checks and steps
[expand] expanding /Users/user/github/CppCMakeVcpkgTemplate
[cmakefileapi-driver] Configuring using preset
[cmakefileapi-driver] Invoking CMake /opt/homebrew/bin/cmake with arguments ["-DCMAKE_TOOLCHAIN_FILE=/Users/user/github/CppCMakeVcpkgTemplate/vcpkg/scripts/buildsystems/vcpkg.cmake","-S/Users/user/github/CppCMakeVcpkgTemplate","-B/Users/user/github/CppCMakeVcpkgTemplate/builds/ninja-multi-vcpkg","-G","Ninja Multi-Config"]
[proc] Executing command: /opt/homebrew/bin/cmake -DCMAKE_TOOLCHAIN_FILE=/Users/user/github/CppCMakeVcpkgTemplate/vcpkg/scripts/buildsystems/vcpkg.cmake -S/Users/user/github/CppCMakeVcpkgTemplate -B/Users/user/github/CppCMakeVcpkgTemplate/builds/ninja-multi-vcpkg -G "Ninja Multi-Config"
```

I am using cmake `v3.27.7`.

**Review by lukka — CHANGES_REQUESTED** [review]
please drop only the schema attribute, and keep the toolchainFile one.

**Comment by lukka** [comment]
This PR has been superseded by https://github.com/lukka/CppCMakeVcpkgTemplate/pull/25

**Pull request closed without merging**